menu/reader.js

```diff
@@ -0,0 +1,41 @@
+const fs = require("fs");
+const YAML = require("js-yaml");
+
+class Menu {
+    constructor(text) {
+        this.delivery = text['Delivery'];
+        this.instructions = text['Instructions']; // If instructions provided
+        this.dishes = text['Menu'];
+        this.modifiers = text['Modifiers'];
+    }
+}
+
+// Convert .yml file to yml text
+function readYmlFile(filename) {
+    try {
+        const raw = fs.readFileSync(filename, 'utf-8');
+        return raw
+    } catch (e) {
+        console.log(e)
+    }
+}
+
+// Returns menu object
+function getMenu(rawText) {
```

Comment:
since the reading of file (excluding error handling) is a single line, could you just merge it into this function please